packages/client/mutations/AcceptRequestToJoinDomainMutation.ts

```diff
@@ -0,0 +1,57 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {SimpleMutation} from '../types/relayMutations'
+import {AcceptRequestToJoinDomainMutation as TAcceptRequestToJoinDomainMutation} from '../__generated__/AcceptRequestToJoinDomainMutation.graphql'
+import SendClientSegmentEventMutation from './SendClientSegmentEventMutation'
+
+graphql`
+  fragment AcceptRequestToJoinDomainMutation_success on AcceptRequestToJoinDomainSuccess {
+    viewer {
+      ...ReviewRequestToJoinOrgModal_viewer @relay(mask: false)
+    }
+  }
+`
+
+const mutation = graphql`
+  mutation AcceptRequestToJoinDomainMutation($requestId: ID!, $teamIds: [ID!]!) {
+    acceptRequestToJoinDomain(requestId: $requestId, teamIds: $teamIds) {
+      ... on ErrorPayload {
+        error {
+          message
+        }
+      }
+      ...AcceptRequestToJoinDomainMutation_success @relay(mask: false)
+    }
+  }
+`
+
+const AcceptRequestToJoinDomainMutation: SimpleMutation<TAcceptRequestToJoinDomainMutation> = (
+  atmosphere,
+  variables
+) => {
+  return commitMutation<TAcceptRequestToJoinDomainMutation>(atmosphere, {
+    mutation,
+    variables,
+    onCompleted: (res) => {
+      const error = res.acceptRequestToJoinDomain.error
+      if (!error) {
+        atmosphere.eventEmitter.emit('addSnackbar', {
+          message: '🎉 Success! User added',
+          autoDismiss: 5,
+          key: 'acceptRequestToJoinDomainSuccess'
+        })
+        SendClientSegmentEventMutation(atmosphere, 'Join Request Accepted', {
+          teamIds: variables.teamIds
+        })
```

Comment:
+1 Would like to see us use a single "Reviewed" event.
```suggestion
        SendClientSegmentEventMutation(atmosphere, 'Join Request Reviewed', {
          action: 'Accept',
          teamIds: variables.teamIds
        })
```